newIDE/app/src/VariablesList/ObjectGroupVariablesDialog.js

```diff
@@ -0,0 +1,201 @@
+    lastSelectedVariableNodeId.current =
+      nodes.length > 0 ? nodes[nodes.length - 1] : null;
+  }, []);
+
+  const shouldCreateVariable = React.useRef<boolean>(
+    shouldCreateInitiallySelectedVariable || false
+  );
+  const actualInitiallySelectedVariableName = React.useRef<?string>(
+    initiallySelectedVariableName
+  );
+  if (shouldCreateVariable.current) {
+    shouldCreateVariable.current = false;
+    const { name: actualVariableName } = insertInVariablesContainer(
+      groupVariablesContainer.current,
+      initiallySelectedVariableName
+        ? getRootVariableName(initiallySelectedVariableName)
+        : 'Variable',
+      null,
+      groupVariablesContainer.current.count(),
+      null
+    );
+    actualInitiallySelectedVariableName.current = actualVariableName;
+    lastSelectedVariableNodeId.current = getNodeIdFromVariableName(
+      actualVariableName
+    );
+  }
+
+  const { DismissableTutorialMessage } = useDismissableTutorialMessage(
+    'intro-variables'
+  );
+
+  return (
+    <Dialog
+      title={<Trans>{objectGroup.getName()} variables</Trans>}
+      actions={[
+        <FlatButton
+          label={<Trans>Cancel</Trans>}
+          onClick={onCancel}
+          key="Cancel"
+        />,
+        <DialogPrimaryButton
+          label={<Trans>Apply</Trans>}
+          primary
+          onClick={apply}
+          key="Apply"
+          id="apply-button"
+        />,
+      ]}
+      secondaryActions={[
+        hotReloadPreviewButtonProps ? (
+          <HotReloadPreviewButton
+            key="hot-reload-preview-button"
+            {...hotReloadPreviewButtonProps}
+          />
+        ) : null,
+        <HelpButton
+          helpPagePath={'/all-features/variables/object-variables'}
+          key="help"
+        />,
+      ]}
+      onRequestClose={onCancel}
+      onApply={apply}
+      open={open}
+      flexBody
+      fullHeight
+      id="object-group-variables-dialog"
+    >
+      <Column expand noMargin noOverflowParent>
+        {groupVariablesContainer.current.count() > 0 &&
+          DismissableTutorialMessage && (
+            <Line>
+              <Column expand>{DismissableTutorialMessage}</Column>
+            </Line>
+          )}
+        <VariablesList
+          projectScopedContainersAccessor={projectScopedContainersAccessor}
+          variablesContainer={groupVariablesContainer.current}
+          areObjectVariables
+          initiallySelectedVariableName={
+            actualInitiallySelectedVariableName.current
+          }
+          emptyPlaceholderTitle={
+            <Trans>Add your first object group variable</Trans>
+          }
+          emptyPlaceholderDescription={
+            <Trans>
+              These variables hold additional information and are available on
+              all objects of the group.
+            </Trans>
+          }
+          helpPagePath={'/all-features/variables/object-variables'}
```

Comment:
Should this be updated?

Reply:
I think the link should be kept.
I guess that we'll add a note in the `object-variables` pages about "mixed values" and link to a section in the object group page that explains what happens to objects when variables are modified in the group editor (which will link back to the `object-variables` page).